numpy/core/code_generators/ufunc_docstrings.py

```diff
@@ -2860,6 +2862,46 @@ def add_newdoc(place, name, doc):
 
     """)
 
+add_newdoc('numpy.core.umath', 'divmod',
+    """
+    Return element-wise quotient and remainder simultaneously.
+
+    .. versionadded:: 1.13.0
+
+    ``np.divmod(x, y)`` is equivalent to ``(x // y, x % y)``, but faster
+    because it avoids redundant work. It is used to implement the Python
+    built-in function ``divmod`` on NumPy arrays.
+
+    Parameters
+    ----------
+    x1 : array_like
+        Dividend array.
+    x2 : array_like
+        Divisor array.
+    out : tuple of ndarray, optional
+        Arrays into which the output is placed. Their types are preserved and
+        must be of the right shape to hold the output.
+
+    Returns
+    -------
+    out1 : ndarray
+        Element-wise quotient resulting from floor division.
+    out2 : ndarray
+        Element-wise remainder from floor division.
+
+    See Also
+    --------
+    floor_divide : Equivalent of Python ``//`` operator.
+    remainder : Equivalent of Python ``%`` operator.
+    modf : Equivalent to ``divmod(x, 1.0)``.
```

Comment:
The  `modf` function comes from the C library and doesn't agree with the Python divmod for negative floats.
```
In [1]: np.modf(-1.5)
Out[1]: (-0.5, -1.0)

In [2]: divmod(-1.5, 1)
Out[2]: (-2.0, 0.5)
```